core/tests/fork-selection.rs

```diff
@@ -129,7 +129,7 @@ impl Vote {
 }
 
 #[derive(Debug)]
-pub struct LockTower {
+pub struct Tower {
```

Comment:
Do these tests completely bypass the real implementation?

Reply:
it appears so.  I didn't realize.